scripts/build_jswrapper.py

```diff
@@ -313,10 +353,16 @@ def codeOutBuiltins(indent, builtin):
 codeOut('');
 codeOut('');
 
-codeOut('const JswSymList jswSymbolTables[] = {');
+# output the strings, possibly with __attribute__ to put them into flash
```

Comment:
Isn't what I have cleaner? Fewer lines of code and no conditional in the part that humans get to read...